test/reducers/users.spec.js

```diff
@@ -0,0 +1,79 @@
+import deepFreeze from 'deep-freeze';
+import users from '../../app/assets/javascripts/reducers/users';
+import '../testHelper';
+import {
+  RECEIVE_USERS,
+  ADD_USER,
+  REMOVE_USER,
+  SORT_USERS
+} from '../../app/assets/javascripts/constants';
+
+describe('users reducer', () => {
+  it('should return initial state when no action nor state is provided', () => {
+    const newState = users(undefined, { type: null });
+    expect(newState.users).to.be.an('array');
+    expect(newState.isLoaded).to.eq(false);
+    expect(newState.sort).to.be.an('object');
+  });
+
+  it('returns the previous state and updates users array from action via RECEIVE_USERS, ADD_USER and REMOVE_USER', () => {
+    const initialState = { title: 'title', isLoaded: false };
```

Comment:
This isn't the shape of the initial state. If you want a shortcut to it, you can just do `initialState = users(undefined, { type: null });`